docs/developers/bitesize/ci/travis_centaur.md

```diff
@@ -0,0 +1,75 @@
+## Travis builds by category of user
+
+| Backend | Any User | Committers |
+|---------|:--------:|:----------:|
+| AWS     |          |     ✅     |
+| BCS     |          |     ✅     |
+| Local   |    ✅    |     ✅     |
+| PAPI V1 |          |     ✅     |
+| PAPI V2 |          |     ✅     |
+| SLURM   |    ✅    |     ✅     |
+| TES     |    ✅    |     ✅     |
+
+On cloud infrastructures that require secured credentials, tests run for users who have commit access, and are skipped for all other users. Other backends run tests run for any user.
```

Comment:
I'd move this explanation above the table to give readers a little context instead of going straight into the table.